assets/js/blocks/featured-product/block.js

```diff
@@ -412,6 +417,10 @@ const FeaturedProduct = ( {
 			contentAlign !== 'center' && `has-${ contentAlign }-content`
 		);
 
+		const containerStyle = {
+			borderRadius: style.border.radius,
```

Comment:
```suggestion
			borderRadius: style?.border?.radius,
```

Reply:
@sunyatasattva I am still getting a block error in the editor because I think you missed this change.